ixmp/core.py

```diff
@@ -393,6 +393,39 @@ def check_access(self, user, models, access='view'):
         else:
             return {model: result.get(model) == 1 for model in models_list}
 
+    def get_meta(self, *args, **kwargs):
```

Comment:
we can avoid this redundant calls by adding `get_meta` and `set_meta` to `_backend_direct` list

Reply:
see this commit https://github.com/iiasa/ixmp/pull/353/commits/121a876b7dafd5839c9b2a2ad53e40ff22754482